web/src/assets/help/welcome-message.md

```diff
@@ -0,0 +1,7 @@
+Welcome to DashboardHub, the no-setup, open-source platform that makes your deployment pipeline more transparent by ensuring everybody on your team is on the same page. DashboardHub does this by gathering the most up-to-date version control information (like what version is on what stage, build/deployment durations) all in one place, and monitors your deployments every 30 minutes and after each deployment to ensure everything is still up and running.
+
+Please click the key icon on the top right to log in using your GitHub account.
+
+For more detailed help and FAQs, [click here](./web/src/app/help/help.component.html).
+
+The current page is showing you the full list of public environments; to access your private environments, please click the menu button on the top left and select "My Environments".
```

Comment:
I would probably keep it simple for now and put this content in the `mat-card-content` in the card for now 🤓